src/GroupManager.php

```diff
@@ -459,6 +470,12 @@ protected function refreshGroupRelationMap() {
 
     foreach ($this->entityTypeBundleInfo->getAllBundleInfo() as $group_content_entity_type_id => $bundles) {
       foreach ($bundles as $group_content_bundle_id => $bundle_info) {
+
+        if ($group_content_bundle_id == 'user') {
```

Comment:
This seems like a good idea, as in Og8 we indeed no longer treat a user as group content. A user is a special case.

/cc @pfrenssen